frontend/src/modules/search/components/SearchBox.js

```diff
@@ -129,6 +126,12 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
         // $FLOW_IGNORE (errors that I don't understand, no help from the Web)
         document.addEventListener('keydown', this.handleShortcuts);
         this.updateFiltersFromURLParams();
+
+        // On mount, update the search input content based on URL.
```

Comment:
It would be nice to say why this is done outside `updateFiltersFromURLParams()`, otheriwise I'm afraid I see future self restoring the old behaviour by mistake, which is hard to catch (no test). A reference to bug 1611666 would be sufficient.